README.md

```diff
@@ -0,0 +1,29 @@
+# Interpolation Library
+
+The NCEP general interpolation library (iplib) contains Fortran 90 subprograms
+to be used for interpolating between nearly all (rectilinear) grids used at NCEP.
+For more detailed documentation see [README.iplib](README.iplib).
+
+### Prerequisites
+
+Compilers: GNU | Intel | Clang | AppleClang | PGI
+
+
+### Installing
+
+```
+mkdir build
+cd build
+cmake -DCMAKE_INSTALL_PREFIX=/path/to/install /path/to/NCEPLIBS-ip
+make -j2
+make install
+```
+
```

Comment:
```
# Testing

Testing requires [pFUnit](https://github.com/Goddard-Fortran-Ecosystem/pFUnit).
```

```
cmake .. -DENABLE_TESTS=ON -DCMAKE_PREFIX_PATH="/path/to/pfunit /path/to/NCEPLIBS"
make -j2
ctest
```

Reply:
Ok.  That is good.  

There is another readme in the ./tests subdirectory.  You should remove the section on how to run the test (now obsolete) or refer the user to the README.md.

Reply:
Having some trouble building it.  

Another question, does test_mod.pf replace the tests in ./unit_test and ./unit_test_for_c?

Reply:
Yes, `test_mod.pf` (should probably be given a better name) contains the code for each unit test. I re-factored the code from those files and converted it to pfunit. I did the scalar part but not the vector. After I finish the other libraries I'll come back and add the vector tests.